### DISCUSSION TITLE
Branches not getting merged

### DISCUSSION THREAD

**Discussion by nolddor** [renovatebot/renovate#22928] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I dunno why branches are not getting merged, see:
https://github.com/nolddor/steam-badges-db/branches

On Renovate panel i see the job pending forever:
![imagen](https://github.com/renovatebot/renovate/assets/1280022/d415c170-60a1-4b07-b647-e9ab57f026c0)

Any idea why?
Renovate config can be found at **[HERE](https://github.com/nolddor/steam-badges-db/blob/main/renovate.json)**

### Relevant debug logs

N/A

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
Renovate runs daily for any repo which doesn't yet have a merged *Pull Request*

**Reply by nolddor** [reply]
So as you can see in the picture, yesterday run is not there. Today's is still pending, is that expected?

Note that i am not using PR but `:automergeBranch` at config.

**Reply by HonkingGoose** [reply]
Here's their `renovate.json` config file:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:js-app",
    "npm:unpublishSafe",
    ":rebaseStalePrs",
    ":automergeBranch",
    ":automergeAll",
    ":disableDependencyDashboard",
    ":disableRateLimiting",
    ":assignee(nolddor)",
    ":label(🤖 Automatic)",
    ":semanticCommits",
    ":semanticCommitTypeAll(chore)"
  ]
}
```

**Reply by nolddor** [reply]
Just discovered basically are working but really slow...

When the branches are created, renovate do not check for the tests until next 24h execution. If there were new commits instead of merge those branches they have to rebase existing and wait again for next 24h execution...

So it only merge one branch ever 24h if there are not more commits since last renovate execution,... is there any way to tell renovate to go back and check for the test to finish after a couple of hours to raise the new branch creation or to rebase an existing one?

**Reply by viceice** [reply]
use platform automerge
